modules/Model.js

```diff
@@ -1,23 +1,69 @@
 (function(root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define(['underscore', 'backbone', './pollingMixin'], factory);
+    define(['underscore', 'backbone', './pollingMixin', './Events'], factory);
   } else if (typeof exports === 'object') {
-    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'));
+    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'), require('./Events'));
   } else {
     root.Torso = root.Torso || {};
-    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling);
+    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling, root.Torso.Events);
   }
-}(this, function(_, Backbone, pollingMixin) {
+}(this, function(_, Backbone, pollingMixin, Events) {
   'use strict';
 
-  /**
-   * Generic Model
-   * @module    Torso
-   * @class     Model
-   * @constructor
-   * @author kent.willis@vecna.com
-   */
-  var Model = Backbone.Model.extend({});
+
+  var Model = Backbone.Model.extend({
+
+    /**
+     * Overrides constructor to apply timing event triggers to Torso.Events
+     * @method constructor
+     * @override
+     */
+    constructor: function() {
+      Backbone.Model.apply(this,arguments);
+      this.__applyTimingEventListeners(['pre-fetch', 'post-fetch']);
+    },   
+
+    fetch: function(options) {
+      var uuid = _.uniqueId();
```

Comment:
I don't think uniqueId will work because you want unique ids server-side, not client-side, correct?  Unless you uniquify them by adding some kind of unique session id.